src/components/TodoTasks.tsx

```diff
@@ -1,21 +1,23 @@
 import React from 'react'
+import { ToggleComplete } from '../App';
 import { TodoList } from '../interfaces/todo'
 import TodoTask from './TodoTask';
 
-interface Props {
-  todoList: TodoList;
+interface ListProps {
+    todoList: TodoList;
+    toggleComplete: ToggleComplete;
 };
 
-const TodoTasks = ({ todoList }: Props) => {
-  return (
-    <div className="task">
-      {todoList.map((todoTask, idx) => {
-        return (
-          <TodoTask todoTask={todoTask} key={idx} />
-        );
-      })};
-    </div>
-  );
+const TodoTasks = ({ todoList, toggleComplete }: ListProps) => {
+    return (
+        <div className="task">
+            {todoList.map((todoTask, idx) => {
+                return (
+                    <TodoTask todoTask={todoTask} toggleComplete={toggleComplete} key={idx} />
+                );
+            })};
```

Comment:
Please remove semicolon here. This semicolon after curly braces is now in fact a string literal and will be rendered as text.